src/wallet/rpcwallet.cpp

```diff
@@ -3129,7 +3148,7 @@ UniValue z_listaddresses(const UniValue& params, bool fHelp)
     if (fHelp || params.size() > 1)
         throw runtime_error(
             "z_listaddresses ( includeWatchonly )\n"
-            "\nReturns the list of zaddr belonging to the wallet.\n"
+            "\nReturns the list of Sprout and Sapling shielded addresses belonging to the wallet.\n"
```

Comment:
I like the change from "zaddr" to "shielded addresses". Neither here nor there, but we don't really need to say "Sprout and Sapling". Shouldn't that be the default unless we otherwise specify?

Reply:
I just wanted to be clearer for downstream users, given that technically previously they only ever received Sprout addresses from here, whereas now they need to check what type of address they are getting (if they are then doing something Sprout-specific, rather than using our regular APIs for which they will be interchangeable).